business/checkers/destinationrules/disabled_namespacewide_mtls_checker.go

```diff
@@ -19,34 +19,34 @@ func (m DisabledNamespaceWideMTLSChecker) Check() ([]*models.IstioCheck, bool) {
 		return validations, true
 	}
 
-	// otherwise, check among Policies for a rule enabling mTLS
-	for _, mp := range m.MTLSDetails.Policies {
-		if enabled, mode := kubernetes.PolicyHasMTLSEnabled(mp); enabled {
-			// If Policy has mTLS enabled in STRICT mode
+	// otherwise, check among PeerAuthentications for a rule enabling mTLS
+	for _, mp := range m.MTLSDetails.PeerAuthentications {
+		if enabled, mode := kubernetes.PeerAuthnHasMTLSEnabled(mp); enabled {
+			// If PeerAuthn has mTLS enabled in STRICT mode
 			// traffic going through DestinationRule won't work
 			if mode != "PERMISSIVE" {
 				check := models.Build("destinationrules.mtls.policymtlsenabled", "spec/trafficPolicy/tls/mode")
 				return append(validations, &check), false
 			} else {
-				// If Policy has mTLS enabled in PERMISSIVE mode
+				// If PeerAuthn has mTLS enabled in PERMISSIVE mode
 				// traffic going through DestinationRule will work
-				// no need for further analysis in MeshPolicies
+				// no need for further analysis in MeshPeerAuthentications
```

Comment:
Probably I will resolve my question when I read more code, but so, this duality PeerAuthentication and MeshPeerAuthentication will continue in Istio 1.6, isn't it ?

Reply:
There aren't two objects (like Policy and MeshPolicy) but the PeerAuthentications in the istio-system are the ones at mesh level. Therefore, instead of having MeshPeerAuthentication (without namespace) they have PeerAuthentication in istio-system.
A better wording for this would be: `no need for further analysis in mesh-wide PeerAuthentications`